src/components/TodoForm/TodoForm.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './TodoForm.css';
+
+import users from '../../api/users';
+
+export class TodoForm extends React.Component {
+  state = {
+    userName: '',
+    todoTitle: '',
+    showEmptyNameMessage: false,
+    showEmptyTitleMessage: false,
```

Comment:
rename these variables, sounds too complicated (mb isEmptyUser or isEmptyTitle)

Reply:
also let's create an errors fields in state, which will contain all errors as props

```
errors: {
  error 1
  error 2
}
```